recipe/build.sh

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env bash
 set -e
 
-check-glibc lib/$cuda_major/*.so*
+check-glibc lib/*.so*
 
 mkdir -p $PREFIX/include
 mv include/* $PREFIX/include/
 mkdir -p $PREFIX/lib
-mv lib/$cuda_major/*.so* $PREFIX/lib/
+mv lib/*.so* $PREFIX/lib/
```

Comment:
Did the directory structure of the download change?

Reply:
Yes.